test/suite/test_sweep03.py

```diff
@@ -72,18 +90,14 @@ def test_disable_idle_timeout1(self):
 
         #
         # The idle timeout is disabled - we don't expect the sweep server to
-        # close any regular handles. Give the sweep server 5x the scan
-        # interval to close any handles.
+        # close any regular handles.
         #
-        time.sleep(5)
+        self.wait_for_sweep(0)
```

Comment:
The function returns a value but you don't assign anything here. Maybe assign it to `unused` and add a comment.

Reply:
I've updated the comment - I don't think always assigning returns is idiomatic in Python - so I didn't add an unused variable.